pkg/api/customization/workload/workload.go

```diff
@@ -236,6 +252,7 @@ func DeploymentFormatter(apiContext *types.APIContext, resource *types.RawResour
 	resource.Actions["pause"] = apiContext.URLBuilder.ActionLinkByID(workloadSchema, workloadID, "pause")
 	resource.Actions["resume"] = apiContext.URLBuilder.ActionLinkByID(workloadSchema, workloadID, "resume")
 	resource.Actions["rollback"] = apiContext.URLBuilder.ActionLinkByID(workloadSchema, workloadID, "rollback")
+	resource.Actions["redeploy"] = apiContext.URLBuilder.ActionLinkByID(workloadSchema, workloadID, "redeploy")
```

Comment:
This only applies to deployments. We can also redeploy a statefulset/daemonset/replicaset in the UI. Can you check the complete list and make it work for all those types?